js/assessor.js

```diff
@@ -152,4 +152,17 @@ Assessor.prototype.calculateOverallScore  = function() {
 	return Math.round( totalScore / ( results.length * ScoreRating ) * 100 );
 };
 
+/**
+ * Register an assessment for a specific plugin
```

Comment:
The assessor knows nothing about plugins, it will just add an assessment to its internal assessment object.